gluecodium/src/main/resources/templates/dart/DartProperty.mustache

```diff
@@ -28,7 +28,8 @@
 }}{{>dart/DartFunctionBody}}
 {{/getter}}
 {{#setter}}
-{{#unless skipDocs}}{{>dart/DartDocumentation}}
+{{#unless skipDocs}}{{>dart/DartFunctionDocs}}
+
```

Comment:
This blank line is extraneous, please remove it.

Reply:
This blank line is added purposely.
Without this line the output result was like:
```bash
/// [value] Some very useful property.
///set isSomeProperty(bool value) {
  ```
It was commenting the actual function definition line.